webapp/src/pages/createApplicationForm/components/Section.js

```diff
@@ -370,15 +340,16 @@ export const Section = forwardRef(({
               <i className="fas fa-angle-down fa-section fa-duplicate"></i>
               {t("Move Section Down")}
             </button>
-            <button
-              className="dropdown-item delete-section"
-              onClick={handleKey}
-            >
-              {isKeyOn
-                && <i className="fas fa-check fa-duplicate fa-section"></i>
-              }
-              {t("Add Key")}
-            </button>
+            {!isReview && (
```

Comment:
Would prefer to have explicit feature flags like "hasKey" has "hasDependency" etc rather than using isReview, again because there might be a different set of features used in a third form type

Reply:
Done! You can review again @avishkar58